### PR TITLE
Update smarts string for ester groups

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:

Update smarts string for `opls_468` to be more specific (related to #427). Also update reference for `opls_465` to `opls_469`. Will also try to add a test case for this in a later commit. Also add smarts string for `opls_490` (ethoxy carbon)

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [x] Appropriate docstring(s) are added/updated
 - [x] Code is (approximately) PEP8 compliant
 - [x] Issue(s) raised/addressed?

### CODE REVIEW

**Review by justinGilmer — COMMENTED** [review]
@daico007 and I worked through this together, once all tests pass and we have an additional molecule or two to test, this is G2G for me!

**Comment by daico007** [comment]
i think this PR should have addressed the atomtying issues for pegda molecule (charge + specific hydrogens). Pending test and this should be ready to go.

**Review by justinGilmer — CHANGES_REQUESTED** [review]
Small change, and then a recommendation to change `opls_182` def from:

`[C;X4]([O;%opls_180])(H)(H)`
to
`[C;X4]([O;%opls_180])(H)(H)C`

Since the description says: `desc="C(H2OR), methyl ether"`

File: foyer/forcefields/xml/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 v0.0.1 - April 15, 2021
  - started versioning of forcefield xmls
-
+v0.0.2 - June 24, 2021
+ - update SMARTS string for opls_468
+ - update references for opls_465 to opls_469
```
[the remark this answers is not in the document]
Comment (daico007) [reply foyer/forcefields/xml/CHANGELOG.md:12]: done!

File: foyer/forcefields/xml/CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,10 @@
 v0.0.1 - April 15, 2021
  - started versioning of forcefield xmls
 v0.0.2 - June 24, 2021
- - update SMARTS string for opls_468
+ - update SMARTS string for opls_180 (from `[C;X4]([O;%opls_180])(H)(H)` to `[C;X4]([O;%opls_180])(H)(H)C`)
+ - update SMARTS string for opls_468 (from `[C;X4]([O;%opls_467])(H)(H)` to `[C;X4]([O;%opls_467])(H)(H)H`)
+ - update SMARTS string for opls_469 (from `H[C;%opls_468]` to `H[C;%opls_468,%opls_490]`)
+ - add SMARTS string for opls_490 (`[C;X4]([O;%opls_467])(H)(H)C`; ethoxy carbon
  - update references for opls_465 to opls_469
```
Comment (justinGilmer) [inline foyer/forcefields/xml/CHANGELOG.md:12]: One additional thing, let’s list these out as well. I think for most changes we make, we should list each affected atomtype as well. Thanks for already adding in the other changes I requested!

**Review by justinGilmer — CHANGES_REQUESTED** [review]
One additional change for the change log

File: foyer/forcefields/xml/CHANGELOG.md (hunk 2 above)
Comment (daico007) [reply foyer/forcefields/xml/CHANGELOG.md:12]: by affected atomtype, do you mean those new overrides?

Comment (justinGilmer) [reply foyer/forcefields/xml/CHANGELOG.md]: Yes, but also opls 465 to 469 should all be listed, since each of them changed their citation.

Comment (daico007) [reply foyer/forcefields/xml/CHANGELOG.md]: done

**Review by justinGilmer — APPROVED** [review]
Confirmed the TOP and GRO files manually with @daico007 .

**Comment by justinGilmer** [comment]
Actually, @daico007  there are 2 tests are failing due to version number changes. After that, its g2g

**Pull request merged**